src/bin/glif_editor.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 the Runebender authors.
+
+//! A quick demonstration of loading and displaying a UFO glyph.
+
+use norad::glyph::Glyph;
+
+use druid_shell::platform::WindowBuilder;
+use druid_shell::win_main;
+
+use druid::{UiMain, UiState};
+
+use runebender::widgets::GlyphEditor;
+
+fn build_ui(ui: &mut UiState, glyph: Glyph) {
+    let root_id = GlyphEditor::new(glyph).ui(ui);
+    ui.set_focus(Some(root_id));
+    ui.set_root(root_id);
+}
+
+fn main() {
+    let glyph_path = match std::env::args().skip(1).next() {
+        Some(arg) => arg,
+        None => {
+            eprintln!("Please pass a path to a .glif file");
+            std::process::exit(1);
+        }
+    };
+
+    println!("loading {}", glyph_path);
```

Comment:
A note, this is a good use case for the new [`dbg!`](https://doc.rust-lang.org/std/macro.dbg.html) macro (1.32+), though I certainly am still in the println habit.

Reply:
yea, I often find it's too verbose, and I prefer just hand-writing 😒